io.md

```diff
@@ -862,13 +862,13 @@ func TestTape_Write(t *testing.T) {
     tape_test.go:23: got 'abc45' want 'abc'
 ```
 
-As we thought! It simply writes the data we want, leaving over the rest.
+As we thought! It writes the data we want, but leaves the rest of the original data remaining.
```

Comment:
Apparently "leaving over" is a British-ism. We have "leftover" in American English, but not typically the verb "leaving over".

Reply:
I get the feeling this probably wont be the only British-ism in the book and I'd prefer to keep it consistently British English if I can; unless it's very unclear to a more international audience. Give me a steer, was it unclear you you what I meant? If so I'm happy to change it.

Reply:
I asked my (American) partner, and he didn't understand it at first. When I explained the situation, he worded it as "not touching the rest", which might be a better reword than my suggestion. I'm nearly through every tutorial, and I think this was the only phrasing I was unfamiliar with. The British spellings are easy to parse, but this phrase jarred me enough to look it up.